### DISCUSSION TITLE
Undefined variable: bulkActions ?

### DISCUSSION THREAD

**Discussion by Mz-DucPN** [livewire/livewire#3939] [1. Help] [answered]
<img width="1207" alt="Screen Shot 2021-10-04 at 10 56 25" src="https://user-images.githubusercontent.com/86099415/135790548-1a99f5c7-69c4-4126-9d92-6a95b3338969.png">
error after require ckfinder/ckfinder-laravel-package, please help

**Comment by joshhanley** [comment, marked as answer]
@Mz-DucPN that looks like an issue with the livewire-datatables package you are using. Can I suggest raising an issue on their repository?

Thanks!
